cmd/utils/git.go

```diff
@@ -88,11 +90,16 @@ func IsCleanDirectory(ctx context.Context, path string) (bool, error) {
 	return status.IsClean(), nil
 }
 
-func GetRandomSHA(ctx context.Context, path string) string {
+// GetRandomSHA returns a random sha generated in the fly
+func GetRandomSHA() string {
 	var letters = []rune("0123456789abcdef")
 	b := make([]rune, 40)
 	for i := range b {
-		b[i] = letters[rand.Intn(len(letters))]
+		n, err := rand.Int(rand.Reader, big.NewInt(int64(len(letters))))
+		if err != nil {
+			oktetoLog.Info("could not generate new int")
```

Comment:
if `err != nil` what is `n` ? should we continue the for loop here? if not, will the `n.Int64()` panic?

Reply:
if `err != nil` then `n = strings.Repeat("0", 40)`